### ISSUE TITLE
A question about making csv

### ISSUE THREAD

**Issue by JunoChann** [sz389/Speech_Recognition#1]
I met a problem when I try to make a csv file of disease classification. It says 
'label'

The above exception was the direct cause of the following exception:

  File "D:\Capstone-Group5-main\Code\Disease_Classification\Data_Preprocessing\generate_csv.py", line 35, in preprocess_Parkinson_csv
    train_df, test_df = train_test_split(df, test_size=0.2, random_state=42, stratify=df["label"])
I'm wondering if you met this problem before.
